src/server/controllers/api/admin/dummies.js

```diff
@@ -3,7 +3,7 @@ import { Dummy } from '@/models'
 
 export class Dummies extends ModelController {
   modelClass = Dummy
-  eagerScope = 'admin'
+  scope = 'admin'
```

Comment:
The concept of `eagerScope` actually translates to `'^admin'` now. I can explain this on slack. already done this today, so will close the PR.